docs/platforms/javascript/common/opentelemetry/index.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: OpenTelemetry Support
+description: "Using OpenTelemetry with Sentry."
+sidebar_order: 7000
+supported:
+  - javascript.nextjs
+  - javascript.node
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.connect
+  - javascript.express
+  - javascript.fastify
+  - javascript.gcp-functions
+  - javascript.hapi
+  - javascript.koa
+  - javascript.nestjs
+  - javascript.sveltekit
+  - javascript.astro
+  - javascript.remix
+notSupported:
+  - javascript
+---
+
+The Sentry SDK uses [OpenTelemetry](https://opentelemetry.io/) under the hood. This means that any OpenTelemetry instrumentation that emits spans will automatically be picked up by Sentry - no further configuration needed!
+
+To start capturing traces and spans emitted from OpenTelemetry, set up <PlatformLink to='/tracing'>Tracing and Performance Monitoring with your Sentry SDK</PlatformLink>. If you do not use tracing, Sentry still connects to OpenTelemetry under the hood to ensure that context isolation and trace propagation works correctly.
```

Comment:
l: sorry for the nit; I think our docs guidelines somewhere tell us to use "don't" over "do not"  

```suggestion
To start capturing traces and spans emitted from OpenTelemetry, set up <PlatformLink to='/tracing'>Tracing and Performance Monitoring with your Sentry SDK</PlatformLink>. If you don't use tracing, Sentry still connects to OpenTelemetry under the hood to ensure that context isolation and trace propagation works correctly.
```

Reply:
yeah totally correct, thank you for checking everything :D